ci/pipeline.yml

```diff
@@ -8,39 +8,75 @@ anchors:
     AWS_ACCESS_KEY_ID: ((aws-access-key-id))
     AWS_SECRET_ACCESS_KEY: ((aws-secret-access-key))
   bosh-creds: &bosh-creds
-    jumpbox_key: |
-      ((jumpbox-key.private_key))
-    ca_cert: |
-      ((ca-cert.certificate))
+    jumpbox_key: ((jumpbox-key))
+    ca_cert: ((ca-cert))
     BOSH_ENVIRONMENT: ((bosh-environment))
 
 resources:
-  - name: kafka-repo
+  - name: boshcli2
+    type: registry-image
+    source:
+      repository: afamuzoka/boshcli2
+      username: ((docker.username))
+      password: ((docker.password))
+      tag: latest
+
+  - name: version
+    type: semver
+    source:
+      driver: s3
+      << : *smarsh_releases_bucket
+      region: eu-west-2
+      key: sredataservices/((bosh-release-name))/version
+      initial_version: ((initial_version))
+
+  - name: git-repo
     type: git
     icon: github-circle
     source:
-      uri: https://github.com/Smarsh/cf-kafka-boshrelease
-      branch: master
+      uri: ((git.url))
```

Comment:
Where are these vars defined? Is the branch value "master"?

Reply:
This was set via `set pipeline` in fly.  If we need to re-factor to hardcode we can it's not a blocker to merging this PR.